spring-ai/src/test/resources/application.yml

```diff
@@ -0,0 +1,3 @@
+spring:
+  ai:
+    openai.api-key: "mock-key-token"
```

Comment:
minor: aren't we missing an empty line after? for lint verification purposes.. not sure though